### DISCUSSION TITLE
Servings per recipe

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1221] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
A simple field like the cooking time for the user to enter the intended number of servings for the recipe

**(Optional): Suggest A Solution**

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

![image](https://user-images.githubusercontent.com/5179945/168413473-e9837f62-375a-45f3-8b76-263deef02bd5.png)

**Comment by hay-kot** [comment]
Doesn't this exists? 

![CleanShot 2022-05-13 at 22 20 44](https://user-images.githubusercontent.com/64056131/168413559-022a9770-3342-4eaf-8226-9af7bf0f7515.png)

![CleanShot 2022-05-13 at 22 21 14](https://user-images.githubusercontent.com/64056131/168413570-e02fc121-3068-46fd-8f96-317d7b38e609.png)

**Comment by terrapinlogo** [comment]
My bad, it does exist sorry about that.
Would the edit field be better placed above the description?
![image](https://user-images.githubusercontent.com/5179945/168414128-ded8e9f2-f5d4-4eaf-b90e-12d65d8696df.png)
and the display next to the cook time with a label?
![image](https://user-images.githubusercontent.com/5179945/168414238-72e4ca9a-ce78-4d09-a666-9e0edbc52473.png)

I looked and found the scaling on the shopping list and the cook page but not the servings field.

**Comment by michael-genson** [comment]
Recipe scaling is available now: https://github.com/mealie-recipes/mealie/issues/507
